Core/Libraries/Source/WWVegas/WW3D2/animatedsoundmgr.cpp

```diff
@@ -158,7 +158,7 @@ Build_List_From_String
 				// Copy this entry into its own string
 				//
 				StringClass entry_string = entry;
-				char *delim_start = ::strstr (entry_string.Peek_Buffer(), delimiter);				
+				char *delim_start = ::strstr (entry_string.Peek_Buffer(), delimiter);
```

Comment:
Minor: This change now contains a few lines were just whitespace was changed. It does not strictly belong into this change. Can we revert this?